src/components/create-challenge-dialog.tsx

```diff
@@ -1,100 +1,70 @@
+'use client'
```

Comment:
Not sure if we want to support custom prompts, but for now this has been condensed to a single form that accepts a users:

name
email
job title
resume
job description